### PR TITLE
[UI] EVEREST-1266-components-page-statuses: fix of components status

### PR DESCRIPTION
**Pull request by solovevayaroslavna**
image

### CODE REVIEW

**Comment by recharte** [comment]
@solovevayaroslavna I didn't review the code but looking at the screenshot in the PR's description it looks like you missed adding the boolean string to the ready column of the containers' rows. See the mock-up in [this ticket](https://perconadev.atlassian.net/browse/EVEREST-1268)

**Comment by recharte** [comment]
Also the name of the container now should be under the name column and we should show the status of the container in the status column. Again this is the scope of https://perconadev.atlassian.net/browse/EVEREST-1268

**Comment by solovevayaroslavna** [comment]
Thanks @recharte! I focused on the statuses, since the task description was about them, so I didn't notice🤦‍♀️ that some of the columns have an incorrect location, I'll fix it and double check it again, thanks again!

**Comment by recharte** [comment]
> Thanks @recharte! I focused on the statuses, since the task description was about them, so I didn't notice🤦‍♀️ that some of the columns have an incorrect location, I'll fix it and double check it again, thanks again!

That's fine because you did focus on EVEREST-1266 and you successfully fixed it. I'm just letting you know that EVEREST-1268 is related and it details some more changes but it can be done in a separate PR.

File: ui/apps/everest/src/pages/db-cluster-details/components/components.constants.ts [hunk 1]
```diff
@@ -22,11 +22,23 @@ export const COMPONENT_STATUS_TO_BASE_STATUS: Record<
...
+export const componentStatusToBaseStatus = (
+  ready: string
+): Record<COMPONENT_STATUS, BaseStatus> => {
+  const arr = ready ? ready.split('/') : [];
+  if (arr.length === 2 && arr[0] === arr[1]) {
+    return {
+      ...COMPONENT_STATUS_TO_BASE_STATUS,
+      [COMPONENT_STATUS.RUNNING]: 'success',
+    };
+  } else return COMPONENT_STATUS_TO_BASE_STATUS;
```
Comment (fabio-silva) [inline ui/apps/everest/src/pages/db-cluster-details/components/components.constants.ts:39]: missing brackets

**Review by fabio-silva — CHANGES_REQUESTED** [review]

**Review by dianabirs — APPROVED** [review]

**Review by fabio-silva — APPROVED** [review]

**Pull request merged**